SMLHelper/Options/ModOptions.cs

```diff
@@ -312,4 +427,48 @@ internal ModChoiceOption(string id, string label, string[] options, int index) :
             Index = index;
         }
     }
+
+    /// <summary>
+    /// A mod option class for handling an option that is a keybind.
+    /// </summary>
+    public class ModKeybindOption : ModOption
+    {
+        public KeyCode Key { get; }
+        public GameInput.Device Device { get; }
+
+        /// <summary>
+        /// Instantiates a new <see cref="ModKeybindOption"/> for handling an option that is a keybind.
+        /// </summary>
+        /// <param name="id">The internal ID of this option.</param>
+        /// <param name="label">The display text to show on the in-game menus.</param>
+        /// <param name="device">The device name.</param>
+        /// <param name="key">The starting keybind value.</param>
+        internal ModKeybindOption(string id, string label, GameInput.Device device, KeyCode key) : base(ModOptionType.Keybind, label, id)
+        {
+            Device = device;
+            Key = key;
+        }
+    }
+
+    /// <summary>
+    /// A mod option class for handling an option that can select one item from a list of values.
+    /// </summary>
+    public class ModDropdownOption : ModOption
+    {
+        public string[] Options { get; }
+        public int Index { get; }
+
+        /// <summary>
+        /// Instantiates a new <see cref="ModDropdownOption"/> for handling an option that can select one item from a list of values.
+        /// </summary>
+        /// <param name="id">The internal ID of this option.</param>
+        /// <param name="label">The display text to show on the in-game menus.</param>
+        /// <param name="options">The collection of available values.</param>
+        /// <param name="index">The starting value.</param>
+        internal ModDropdownOption(string id, string label, string[] options, int index) : base(ModOptionType.Dropdown, label, id)
+        {
+            Options = options;
+            Index = index;
+        }
+    }
```

Comment:
This file is starting to get big, with too many barely related things contained.  
Let's split this up into distinct files for each class.

Reply:
Addressed in commit 20c314b